tools/dev_scripts/ankaios-secure-start

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+set -e
+
+SCRIPT_DIR="$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
+LOG_DIR="$(readlink -f "$SCRIPT_DIR/../../logs")"
+CERTS_FOLDER_NAME=".certs"
+CERTS_DIR="$SCRIPT_DIR/../../$CERTS_FOLDER_NAME"
+
+mkdir -p "$LOG_DIR"
+
+"$SCRIPT_DIR/../certs/create_certs.sh"
+
+echo "Staring up Ankaios server with TLS enabled"
+ANKSERVER_CA_PEM="$CERTS_DIR/ca.pem" \
+ANKSERVER_CRT_PEM="$CERTS_DIR/server.pem" \
+ANKSERVER_KEY_PEM="$CERTS_DIR/server-key.pem" \
+RUST_LOG=debug /workspaces/ankaios/target/x86_64-unknown-linux-musl/debug/ank-server 2> "$LOG_DIR/ank-server" &
+
+echo "Staring up Ankaios agent with TLS enabled"
+ANKAGENT_CA_PEM="$CERTS_DIR/ca.pem" \
+ANKAGENT_CRT_PEM="$CERTS_DIR/agent.pem" \
+ANKAGENT_KEY_PEM="$CERTS_DIR/agent-key.pem" \
+RUST_LOG=debug /workspaces/ankaios/target/x86_64-unknown-linux-musl/debug/ank-agent --name agent_A 2> "$LOG_DIR/ank-agent" &
+
+RED='\033[0;31m'
+NC='\033[0m' # No Color
+
+echo -e "${RED}To use certificates for the ank cli run the cli as follows:${NC}"
+echo 'ank --ca_pem /workspaces/ankaios/.certs/ca.pem --crt_pem /workspaces/ankaios/.certs/cli.pem --key_pem /workspaces/ankaios/.certs/cli-key.pem get workloads'
```

Comment:
Use `$CERTS_DIR` here.
```suggestion
echo "ank --ca_pem \"$CERTS_DIR/ca.pem\" --crt_pem \"$CERTS_DIR/cli.pem\" --key_pem \"$CERTS_DIR/cli-key.pem\" get workloads"
```

Reply:
This was done explicitly, as I didn't want to print out the CERTS_DIR as `CERTS_DIR="$SCRIPT_DIR/../../$CERTS_FOLDER_NAME"` with some `/../../` in the middle ... 
For a dev script, I think the current behavior is fine.